modules/op-starting-pipelines.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-starting-pipelines_{context}"]
+= Starting Pipelines
+
+After you create a Pipeline, you need to start it for the Pipeline to run and execute its Tasks. You can start a Pipeline Run from the *Pipelines* view, *Pipeline Details* page, or the *Topology* view.
+
+.Procedure
+To start a Pipeline using the *Pipelines* view:
+
+. In the *Pipelines* view of the *Developer* perspective, click the *Options* {Kebab} menu adjoining a Pipeline, and select *Start*.
+. The *Start Pipeline* dialog box displays the *Git Resources* and the *Image Resources* based on the Pipeline definition.
++
+[NOTE]
+====
+For Pipelines created using the *From Git* option, the *Start Pipeline* dialog box displays an `APP_NAME` field in the *Parameters* section, in addition. All the fields in the dialog box are prepopulated by the Pipeline templates in the *From Git* flow.
+====
++
+.. If you have resources in your namespace, the *Git Resources* and the *Image Resources* fields are prepopulated with those resources. If required, use the drop-downs to select or create the required resources and customize the Pipeline Run instance.
+.. If required, modify the *Advanced Options* and add credentials to authenticate the specified Git server or Docker registry.
+
+. Click *Start* to start the PipelineRun.
+. The *Pipeline Run Details* page displays the Pipeline being executed. After the Pipeline starts, the Tasks and steps within each Task are executed.
```

Comment:
s/steps/Steps.
Since Step in a concept, its with capital `S`, see About Tasks topic.  Please change the occurrences of step to Step wherever we refer to Step inside a Task. 

There is one place in the "Understanding Pipelines" section, where step is used, can you change it as part of this PR

Reply:
Done, it was in the concepts module.

Reply:
Thank you!